README.md

```diff
@@ -1,36 +1,53 @@
 # OpenCog Hyperon - Distributed Atomspace (DAS)
 
-Here, we keep stuff that is pertinent to the DAS project but doesn't belong to
-any of the other project's specific repositories, such as automated deployment
-scripts, API and high level documentation, assets used by other repositories'
-CI/CD scripts, etc.
+This repository contains items relevant to the DAS project that do not belong in any other project's specific repositories, such as automated deployment scripts, API and high-level documentation, and assets used by other repositories' CI/CD scripts.
 
-## Documentation
+## **Components**
 
-* [DAS API](https://singnet.github.io/das-query-engine/api/DAS/) - Documentation of DAS API
-* DAS User's Guide ([Jupyter Notebook](notebooks/das-users-guide.ipynb)) ([Markdown](docs/das-users-guide.md)) - Document with code snippets showing how to use DAS.
-* [DAS Overview](docs/das-overview.md) - Description of main DAS components and deployment architecture.
-* [DAS Toolbox User's Guide](https://github.com/singnet/das-toolbox) - A guide with detailed instructions to use the DAS Server Toolbox.
-* [Release Notes](docs/release-notes.md) - Release notes of all DAS components.
+DAS consists of several components. Below is a list of these components:
+
+| **Component**             | **Description**                                                                 | **Documentation Link**                                             |
```

Comment:
I don't think you should use a table here. My suggestion is to use a plain list. Something like:

```
* **Query Agent** ([documentation](query_agent/python/README.md)) - A data manipulation API for...
* **Attention Broker** ([documentation](attention_broker/README.md)) - DAS component that keeps...
...
```